test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -4033,7 +4033,7 @@ BOOST_AUTO_TEST_CASE(varM_disqualified_as_keyword)
 			}
 		}
 	)";
-	BOOST_CHECK(!success(text));
+	CHECK_ERROR(text, DeclarationError, "Identifier not found or not unique.");
```

Comment:
This is what it currently outputs, but shouldn't it be a parser error?

Reply:
`uintM` is not a keyword, so it parses it as the name of a type. The error class is fine I woulds ay.

Reply:
Ah right. I just would rather not have native types in the parser to avoid confusion.